ui/src/app/views/resource-card/component.tsx

```diff
@@ -380,6 +398,14 @@ export class ResourceCard extends React.Component<ResourceCardProps, ResourceCar
           <Button onClick={() => this.setState({invalidResourceError: false})}>OK</Button>
         </ModalFooter>
       </Modal>}
+      {this.state.showErrorModal &&
```

Comment:
This feels like a useful reusable component to me. Do you think it would be valuable to put this in `components/modals.tsx`?